packaging/opensuse/snapd.spec

```diff
@@ -15,6 +16,19 @@
 
 %bcond_with testkeys
 
+# Enable AppArmor on openSUSE Tumbleweed (post 15.0) or higher
+%if 0%{?suse_version} >= 1550
```

Comment:
Had to double check the version macros https://en.opensuse.org/openSUSE:Packaging_for_Leap#RPM_Distro_Version_Macros

Reply:
What is the version macro for tumbleweed? Is it always increasing? E.g. once Leap 16 is out will tumbleweed move to 1650?

Reply:
hmm there's a note:
> never rely on this version, it may change

Reply:
It will increase to something, yes. It might be 1660, hell if I know. Basically, just rely on `%suse_version` to be a checkpoint/milestone and you'll be fine.